### PR TITLE
made playlist-songs on PlaylistPage.

### PR DESCRIPTION
**Pull request by Klimpavlov**

### CODE REVIEW

File: PartyPlaylist/HomePage/spotifyClone/Playlist.html [hunk 1]
```diff
@@ -57,20 +57,86 @@ <h5 class="h5-img-text">Klim</h5>
...
+                <td>1</td>
+                <td class="song-title">
+                    <div class="song-image">
+                        <img src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse3.mm.bing.net%2Fth%3Fid%3DOIP.jdWn8XjWxXUW8fTj5Wfe2wHaHa%26pid%3DApi&f=1&ipt=a1599e55ffbca16fd34a1b8d5ccd74811dbf4f675c6c44f99b5e035942f27f41&ipo=images" alt="">
+                    </div>
+                    <div class="song-name-artist">
+                        <div class="song-name">
+                            Mockingbird
+                        </div>
+                        <div class="song-artist">Eminem</div>
```
Comment (AlesBan) [inline PartyPlaylist/HomePage/spotifyClone/Playlist.html:85]: why are there two eminems?
leave pls song-name and song-artist collumns (like on photo)

File: PartyPlaylist/HomePage/spotifyClone/SearchPage.html [hunk 2]
```diff
@@ -69,139 +69,275 @@
     </div>
 </div>
 
+    <!--
+   <section class="section">
+       <table class="table">
+           <tr>
+               <th><h2 class="section-title">Треки</h2></th>
+               <th><h4 class="popularity">Популярность</h4></th>
```
Comment (AlesBan) [inline PartyPlaylist/HomePage/spotifyClone/SearchPage.html:77]: I think we should smth with name "Популярность" maybe some simonim. It is very long and in table don't look very good
Or change everything to english

File: PartyPlaylist/HomePage/spotifyClone/SearchPage.html [hunk 3]
```diff
@@ -69,139 +69,275 @@
...
+    <!--
+    <div class="playlist-artists">
+        <p class="artists">Исполнители</p>
+        <div class="playlist-artists-container">
+            <div class="artist-container">
+                <div class="artist-img">
+                    <img src="https://nownownow.ru/wp-content/uploads/2022/08/90180023-scaled-e1661173059329.jpg" alt="">
+                </div>
+                <div class="artist-name">Boulevard Depo</div>
+                <div class="artist-musician">Исполнитель</div>
```
Comment (AlesBan) [inline PartyPlaylist/HomePage/spotifyClone/SearchPage.html:192]: this div useless
we have <p class="artists">Исполнители</p>

File: PartyPlaylist/HomePage/spotifyClone/SearchPage.html [hunk 4]
```diff
@@ -69,139 +69,275 @@
     </div>
 </div>
 
+    <!--
+   <section class="section">
+       <table class="table">
+           <tr>
```
Comment (AlesBan) [inline PartyPlaylist/HomePage/spotifyClone/SearchPage.html:75]: add pls clock image on duration column like in playlist page

Comment (Klimpavlov) [reply PartyPlaylist/HomePage/spotifyClone/SearchPage.html:75]: oke, i'll fix all that was mentioned

File: PartyPlaylist/HomePage/spotifyClone/Playlist.css [hunk 5]
```diff
@@ -60,3 +57,67 @@
...
+}
+
+.song-name-artist {
+    padding-left: 15px;
+}
+
+.song-artist, .song-album, .song-popularity, tr td:first-of-type, .song-duration, .song-redactor {
+    font-size: 14px;
+    margin-top: 6px;
+    color: white;
```
Comment (AlesBan) [inline PartyPlaylist/HomePage/spotifyClone/Playlist.css:112]: make pls table items color darker than column names color

**Review by AlesBan — APPROVED** [review]

**Pull request merged**